res/controllers/Akai-APC-MINI.midi.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset mixxxVersion="2.0.0+" schemaVersion="1">
+    <info>
+        <name>APC Mini Mapping</name>
+        <author>Valerio Vitali</author>
+        <description>MIDI-input and MIDI-output for light control</description>
```

Comment:
Why light control? This looks like a pad/fader controller.

Reply:
The APC Mini is a 64 button controller for sound effects, so I mapped it to the 64 samplers extended skins that have them.
all the 64 buttons have led that can be controlled, I make them yellow if a sample is configured, green while playing it and red when it stops... very funny :)

Reply:
I guess "light control" can be misinterpreted for some fancy disco light driver... if you have a suggestion for an alternative description, feel free...